src/core_modules/capture-core/components/WidgetEnrollment/WidgetEnrollment.container.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { errorCreator } from 'capture-core-utils';
 import log from 'loglevel';
 import { WidgetEnrollment as WidgetEnrollmentComponent } from './WidgetEnrollment.component';
-import { useOrganizationUnit } from './hooks/useOrganizationUnit';
+import { useOrgUnitName } from '../../metadataRetrieval/orgUnitName';
```

Comment:
I think @JoakimSM was of the opinion that the file which `useOrgUnitName` comes from can be made into a proper dependency.

Reply:
Moving `useOrgUnitName` to a package dependency sounds good to me (perhaps it can be one of the hooks provided by the app runtime library 🤔). To be able to follow up on this, can you create a TECH issue and link it to this ticket? Thank you!